datadryad/dryad-repo
# Doi refactoring

I refactored the code in DOIIdentifierProvider so that the manipulations of dois and metadata happen once in centralized locations. I also added public accessor methods for retrieving dois and versioned dois.

To test:
make package with two files
approve
make version
delete file
note that there is now one file
approve the one-file version
note that there is one file in version .2
make another version
note that there is still one file
add a file
approve the two-file version (might need to redo the search for unclaimed items at workflow overview)
note that there are two files in version .3
